--- /PIGPIO/Makefile.orig	2018-03-20 16:01:29.414738091 +0000
+++ /PIGPIO/Makefile	2018-03-20 16:10:10.086944743 +0000
@@ -92,13 +92,12 @@
 	install -m 0755 pig2vcd           $(DESTDIR)$(bindir)
 	install -m 0755 pigpiod           $(DESTDIR)$(bindir)
 	install -m 0755 pigs              $(DESTDIR)$(bindir)
-	if which python2; then python2 setup.py install; fi
-	if which python3; then python3 setup.py install; fi
+	if which python2; then python2 setup.py install --prefix=$(DESTDIR)$(libdir); fi
+	if which python3; then python3 setup.py install --prefix=$(DESTDIR)$(libdir); fi
 	install -m 0755 -d                $(DESTDIR)$(mandir)/man1
 	install -m 0644 *.1               $(DESTDIR)$(mandir)/man1
 	install -m 0755 -d                $(DESTDIR)$(mandir)/man3
 	install -m 0644 *.3               $(DESTDIR)$(mandir)/man3
-	ldconfig
 
 uninstall:
 	rm -f $(DESTDIR)$(includedir)/pigpio.h
@@ -110,11 +109,10 @@
 	rm -f $(DESTDIR)$(bindir)/pig2vcd
 	rm -f $(DESTDIR)$(bindir)/pigpiod
 	rm -f $(DESTDIR)$(bindir)/pigs
-	if which python2; then python2 setup.py install --record /tmp/pigpio >/dev/null; xargs rm -f < /tmp/pigpio >/dev/null; fi
-	if which python3; then python3 setup.py install --record /tmp/pigpio >/dev/null; xargs rm -f < /tmp/pigpio >/dev/null; fi
+	if which python2; then python2 setup.py install --prefix=$(DESTDIR)$(libdir) --record /tmp/pigpio >/dev/null; xargs rm -f < /tmp/pigpio >/dev/null; fi
+	if which python3; then python3 setup.py install --prefix=$(DESTDIR)$(libdir) --record /tmp/pigpio >/dev/null; xargs rm -f < /tmp/pigpio >/dev/null; fi
 	rm -f $(DESTDIR)$(mandir)/man1/pig*.1
 	rm -f $(DESTDIR)$(mandir)/man3/pig*.3
-	ldconfig
 
 $(LIB1):	$(OBJ1)
 	$(SHLIB) -o $(LIB1) $(OBJ1)
